recipes/diffpy.pdfgui/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "diffpy.pdfgui" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/diffpy/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 4cf3106323544bbd6ead3a774c4dd1db534c618c116e0a0811a33493371150f5
+
+build:
+  # we need egg_dir to avoid overlapping diffpy/__init__.py files.
+  preserve_egg_dir: True
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  entry_points:
+    - pdfgui = diffpy.pdfgui.applications.pdfgui:main
+  osx_is_app: True
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+
+  run:
+    - python >=3.6
+    - setuptools
+    - six
+    - matplotlib-base
+    - wxpython >=4.0.7
+    - menuinst  # [win]
```

Comment:
If this dependency is really necessary we cannot do noarch but it doesn't look like b/c it is not in the metadata.

Reply:
I think these dependency are necessary, as we did before for a conda release, see https://github.com/diffpy/diffpy.pdfgui/blob/v1.4.0/conda-recipe/meta.yaml#L39-L41

Is there any way to do noarch and keep these dependencies?

Reply:
> Is there any way to do noarch and keep these dependencies?

Nope. There is a trick to do a platform noarch but I don't think it is worth in this case. Remove the noarch, add the dependencies and we can merge this.

PS: raise an issue upstream b/c those dependencies are not listed in the package metadata!